App/lib/hooks/usePersistStore.js

```diff
@@ -1,9 +1,9 @@
 import { useEffect } from 'react';
 import AsyncStorage from '@react-native-community/async-storage';
-import { useDispatch, useSelector } from'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 
-import useAppState from './useAppState";
```

Comment:
I don't know how this happened, but I doubt the app would have even run :thinking: